utility/src/main/java/org/oppia/util/logging/EventBundleCreator.kt

```diff
@@ -0,0 +1,84 @@
+package org.oppia.util.logging
+
+import android.os.Bundle
+import org.oppia.app.model.EventLog
+
+const val TIMESTAMP_KEY = "timestamp"
+const val TOPIC_ID_KEY = "topicId"
+const val STORY_ID_KEY = "storyId"
+const val QUESTION_ID_KEY = "questionId"
+const val EXPLORATION_ID_KEY = "explorationId"
+const val PRIORITY_KEY = "priority"
+
+/**
+ * Utility for creating bundles from [EventLog] objects.
+ * Note that this utility may later upload them to remote services.
+ */
+open class EventBundleCreator {
+  private var bundle = Bundle()
+
+  open fun assignBundleValue(eventLog: EventLog): Bundle {
```

Comment:
Can we call this createEventBundle() instead?

Reply:
Done.